src/App.js

```diff
@@ -31,16 +64,55 @@ export default function App() {
     <div className="overflow-y-hidden overflow-x-hidden">
       <GoogleTags />
       <Banner />
-        <Calendar selectedDate={selectedDate} onDateChange={onDateChange} />
+      <div className="flex">
+        <div className="">
           <EventList
             url={`${
               process.env.REACT_APP_API_URL
             }/api/deal?approved.state=active&dayOfWeek=${getDayOfWeek(
               selectedDate.getDay()
-            )}`}
+            )}&${filterResult}${filters.hobo ? '&city=Hoboken' : ''}${filters.jc ? '&city=Jersey City' : ''}${filters.active ? `&active=${filters.active}` : ''}`}
```

Comment:
There's a lot of nuanced logic here for setting the querystring. May be best for us to abstract it and have a create filter url function to craft a url for the specific request. not a blocker but something to definitely iterate on.